File: app/src/main/java/io/github/droidkaigi/confsched2017/api/service/DroidKaigiService.java
package io.github.droidkaigi.confsched2017.api.service;

import java.util.List;

import io.github.droidkaigi.confsched2017.model.Session;
import io.reactivex.Single;
import retrofit2.http.GET;

public interface DroidKaigiService {

    @GET("/sessions.json")
    Single<List<Session>> getSessionsJa();

    @GET("/en/sessions.json")
    Single<List<Session>> getSessionsEn();

}

File: app/src/main/java/io/github/droidkaigi/confsched2017/api/service/DroidKaigiService.kt
package io.github.droidkaigi.confsched2017.api.service

import io.github.droidkaigi.confsched2017.model.Session
import io.reactivex.Single
import retrofit2.http.GET

interface DroidKaigiService {

    @GET("/2017/sessions.json")
    fun getSessionsJa(): Single<List<Session>>

    @GET("/2017/en/sessions.json")
    fun getSessionsEn(): Single<List<Session>>
}
